sound/soc/sof/control.c

```diff
@@ -216,8 +216,7 @@ int snd_sof_bytes_ext_put(struct snd_kcontrol *kcontrol,
 		(const struct snd_ctl_tlv __user *)binary_data;
 	int ret;
 	int err;
-	int max_size = SOF_IPC_MSG_MAX_SIZE -
-		sizeof(const struct sof_ipc_ctrl_data);
+	int max_size = 10 * SOF_IPC_MSG_MAX_SIZE;
```

Comment:
Better to make 10 a SOF macro too!

Reply:
not sure why 10 helps. The TLV can pass 30-50K in one shot, this falls already short of the goal
make that 500K...

Reply:
this was just a quick hack to be able to make the message bigger for testing the actual sending. I'm ok with whatever for max size if someone comes up with good value.

Reply:
@plbossart I now removed the hardcoded value andthe max is coming from topology